lib/modules/datasource/nextcloud/schema.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod';
+import { LooseArray, LooseRecord } from '../../../util/schema-utils';
+
+const Translation = z.object({
+  changelog: z.string(),
+});
+
+export const ApplicationRelease = z.object({
+  created: z.string(),
```

Comment:
If it returns in ISO date format, we could use `zod.iso.date`

Reply:
If I try to replace `z.string()` with `z.iso.date()`, it tells me that the `iso` object does not exist.

I see that zod 3.x is used, and maybe they are only in zod 4.x?

https://zod.dev/api?id=iso-datetimes#iso-dates